package.json

```diff
@@ -110,12 +97,12 @@
     "react-dom": "^17.0.0",
     "react-test-renderer": "16.10.2",
     "rollup": "2.33.1",
-    "rollup-plugin-babel": "4.4.0",
     "rollup-plugin-terser": "5.3.0",
     "rollup-plugin-visualizer": "4.0.4",
     "semver": "7.3.2",
     "styled-components": "4.4.0",
-    "typescript": "4.1.3"
+    "typescript": "4.1.3",
+    "typescript-plugin-styled-components": "1.4.4"
```

Comment:
TODO: This is currently unused. Review the `typescript-plugin-styled-components` docs and call this appropriately.